standards/certification/getting-scs-compatible-certified.md

```diff
@@ -0,0 +1,114 @@
+# Getting SCS-compatible certification (for operators)
+
+## Process overview
+
+The SCS Certification is a technical certification:
+The Operator needs to fulfill technical requirements, such as providing certain
+APIs and guaranteeing certain platform behavior in order to be certifiable.
+
+These requirements are meant to provide guarantees to their customers, allowing
+them to rely on certain features to be available and on certain system behavior
+that lets their applications run in a reliable way.
+
+The SCS certification process typically consists of a few simple steps:
+
+1. Running the SCS compliance test suite and adjusting the infrastructure until it passes.
+2. Any additional declarations (for non-testable aspects) are written and passed to the SCS Forum.
+3. The operator must be a member ("shaper" or "advisor" level) of the Forum SCS-Standards in the
+   OSB Alliance (a non-profit) and pay the respective membership fees. Alternatively fees can
+   be paid without becoming a member.
```

Comment:
Like in my other comment, I would recommend to not hardcode the conditions here unless this becomes the leading document for this.

Reply:
Ideally, we'd have a preliminary Forum Page that we can point to already.
The other option is to sketch the forum conditions here and then remove it as soon as we have the page that we can point to.
Fortunately, the content is not controversial, as this has all been agreed up already thanks to your and Dirk's great work.